src/styles/blocks/wrapper.scss

```
@@ -0,0 +1,7 @@
+.wrapper {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  height: 100vh;
```

A good practice is also to add height: 100dvh, for dynamic height on mobiles (you can read more here https://css-tricks.com/the-large-small-and-dynamic-viewports/). Although dvh is not yet supported in all browsers: https://caniuse.com/?search=dvh but you can write it like 
```suggestion
  height: 100vh;
  height: 100dvh;
```

---

So if 100dvh isn't supported 100vh will be a fallback property?

---

yes exactly :)